torchaudio/transforms.py

```diff
@@ -941,6 +941,12 @@ class ComputeDeltas(torch.nn.Module):
     Args:
         win_length (int, optional): The window length used for computing delta. (Default: ``5``)
         mode (str, optional): Mode parameter passed to padding. (Default: ``'replicate'``)
+
+    Example
+        >>> spectrogram = transforms.Spectrogram()
+        >>> compute_deltas = ComputeDeltas(win_length=5, mode="replicate")
+        >>> original = spectrogram(waveform)
+        >>> delta = compute_deltas(original)
```

Comment:
```suggestion
        >>> specgram = torch.randn(2, 257, 100, dtype=torch.cfloat)
        >>> transform = transforms.ComputeDeltas(win_length=5, mode="replicate")
        >>> delta = transform(specgram)
```

Reply:
@nateanl Thank you for the review! I will make the suggested changes and update the PR.

Reply:
@nateanl I have made the necessary changes. If you want, I can raise a new, clean PR with only the requested changes.

Reply:
Thanks. Let's make a new PR with just one commit then. I will help merge it when you finish.